pack/side/film_fatale_encounter.json

```diff
@@ -0,0 +1,1207 @@
+    "enemy_damage": 1,
+    "enemy_evade": 3,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "health": 5,
+    "illustrator": "Peter Polach",
+    "name": "Saturnite Drudge Militia",
+    "pack_code": "film_fatale",
+    "position": 33,
+    "quantity": 2,
+    "text": "Elusive. Retaliate.\n[action]: <b>Parley.</b> Test [willpower] (3) to make loud noises. For each point you succeed by, deal 1 damage to Saturnite Drudge Militia. If you succeed by 2 or more, exhaust it.",
+    "traits": "Monster. Saturnite.",
+    "type_code": "enemy"
+  },
+  {
+    "clues": 1,
+    "code": "72034",
+    "encounter_code": "film_fatale_space",
+    "encounter_position": 11,
+    "faction_code": "mythos",
+    "illustrator": "Irina French",
+    "name": "Lost Asteroid",
+    "pack_code": "film_fatale",
+    "position": 34,
+    "quantity": 2,
+    "shroud": 4,
+    "text": "<b>Revelation</b> - Disengage from each enemy, put Lost Asteroid into play, and move to it.\nInvestigators at Lost Asteroid get -1 to all skills during the mythos phase.\n[action]: Test [agility] (4). If you succeed, move to a revealed location and discard Lost Asteroid. (You may spend 4 resources to automatically succeed.)",
+    "traits": "Cosmos.",
+    "type_code": "location"
+  },
+  {
+    "code": "72035",
+    "encounter_code": "film_fatale_space",
+    "encounter_position": 13,
+    "faction_code": "mythos",
+    "illustrator": "Rob Laskey",
+    "name": "Celestial Shower",
+    "pack_code": "film_fatale",
+    "position": 35,
+    "quantity": 2,
+    "text": "<b>Revelation</b> - Test [intellect] + [agility] (6). If you fail, investigators at your location take a combined total of X damage, divided as you choose. X is the number of investigators at your location.",
+    "traits": "Hazard.",
+    "type_code": "treachery"
+  },
+  {
+    "back_link": "72036b",
+    "code": "72036",
+    "cost": null,
+    "encounter_code": "film_fatale_space",
+    "encounter_position": 15,
+    "faction_code": "neutral",
+    "flavor": "It seems bigger on the inside.",
+    "illustrator": "Rob Laskey",
+    "name": "Rocket Ship",
+    "pack_code": "film_fatale",
+    "position": 36,
+    "quantity": 1,
+    "subname": "Rattling with Energy",
+    "text": "Vehicle.\n[free] Spend 1 [per_investigator] clues, as a group: Blast off! Move Rocket Ship to a [[Cosmos]] location. Reveal a token from the chaos bag. If a symbol token is revealed, each investigator leaves Rocket Ship. Then, flip this card over.",
+    "traits": "Vehicle. Cosmos.",
+    "type_code": "asset"
+  },
+  {
+    "code": "72036b",
+    "encounter_code": "film_fatale_space",
+    "encounter_position": 15,
+    "enemy_damage": 2,
+    "enemy_evade": 4,
+    "enemy_fight": 4,
+    "enemy_horror": null,
+    "faction_code": "mythos",
+    "flavor": "Your reckless flight has attracted unwanted attention. You can't risk flying until this pesky planetoid is dealt with.",
+    "health": null,
```

Comment:
Unlike `enemy_damage` and `enemy_horror`, this value does need to be explicitly specified as `null` (which you did! Just mentioning.)

Reply:
Yeah my concistency there was down to always using the template, and just keeping null values rather than wiping them. I was aware of the distinction between null as in irrelevant and null as in `-` cannot be defeated in this case.